asdf/asdf.py

```diff
@@ -226,6 +234,15 @@ def extensions(self):
         -------
         list of asdf.extension.ExtensionProxy
         """
+        warnings.warn(
+            "AsdfFile.extensions is deprecated.  Use asdf.get_config().extensions to "
+            "get a list of enabled extensions.",
+            AsdfDeprecationWarning,
```

Comment:
Similar to other suggestions.
```suggestion
            "AsdfFile.extensions is deprecated.  Please see :ref:`extending_extensions_installing_asdf_config`.",
            AsdfDeprecationWarning,
```